CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing
+
+## How to contribute
+
+Contributing to the MSTG can be done in many different ways:
+
+- Create [issues](https://github.com/OWASP/owasp-mstg/issues "MSTG Issues") for missing content or errors. Explain what you think is missing and give a suggestion as to where it could be added.
+- Create a [Pull Request (PR)](https://github.com/OWASP/owasp-mstg/pulls "Create a pull request"). This is a direct contribution to the guide and your PR may be merged after review. Be sure to follow our [style guide](https://github.com/OWASP/owasp-mstg/blob/master/style_guide.md "MSTG Style Guide") when writing content. You should ideally create an issue for any PR you would like to submit, as we can first review the merit of the PR and avoid any unnecessary work. This is of course not needed for small modifications such as correcting typos.
+- Review pull requests. If you are a fluent speaker in any of the different languages that the MSTG is available in, feel free to give feedback on any of the submitted PRs.
+- Proofread and fix errors. If you are studying the MSTG, write down any error, no matter how small, and submit them in an Issue or fix them yourself through a PR.
```

Comment:
We could add the crackmes and playground:

- Extending and maintaining our existing mobile app suite, which is the [Crackmes](https://github.com/OWASP/owasp-mstg/tree/master/Crackmes "Crackmes") and [Hacking Playground](https://github.com/OWASP/MSTG-Hacking-Playground "Hacking Playground")

Reply:
Good point: I wonder, given that we have 3 projects: 
- crackmes+hacking playground 
- MSTG
- MASVS
should we:
1. have a contributing at each and a readme referring to each of the projects?
2. have a contributing at each and a readme + contributing referring to each of the projects?
3. have a contributing at the MASVS/MSTG and have it referring to all 3 projects :) ? @TheDauntless , @sushi2k , @cpholguera : let me know by responding here